kytos/core/interface.py

```diff
@@ -370,31 +297,42 @@ def use_tags(
             tag_type: TAG type value
             use_lock: Boolean to whether use a lock or not
         """
+        if not tags:
+            return
```

Comment:
Added this because of new updates in `mef_eline` but it should be the caller's responsibility to send the correct value types. `[]` or `None` are not part of the supported arguments. Deleted this early return, same case with `make_tags_available()`

Reply:
Yep. Sounds good.